src/mixins/legal-api-mixin.ts

```diff
@@ -100,7 +100,7 @@ export default class LegalApiMixin extends Mixins(FilingTemplateMixin) {
    * Fetches a filing by its id.
    * @returns a promise to return the filing of the specified type
    */
-  async fetchFilingById (id: number): Promise<any> {
```

Comment:
This due to the fact they are all `awaited` where they are invoked?

Reply:
Yes. Methods that return a promise aren't async (sort of). Methods that await something are async.
(Actually I think they're implemented the same under the hood, but I'm trying to follow the syntax that TS wants.)

Reply:
For sure, i like this! I'm certainly guilty of applying `async` to methods where it might be redundant.